README.md

```diff
@@ -89,23 +136,37 @@ Accept: application/fs.api+json
 
 {
   "data": {
-    "action": "Move",
+    "action": "move",
     "parameters": {
-      "destination": "/dogs/"
-    }
+      "destination": "dogs/",
+      "flags": [ "force" ]
+    },
   }
 }
 ```
+
 **[⬆ back to top](#table-of-contents)**
 
 # Response
 
 Similar to the request syntax, every response WILL have a JSON object at the root of the response when it contains data, which defines the response's "top level".
-The only exceptions to this are the `read` and `download` GET requests, which contain ONLY the resource's data.
+The only exceptions to this are the successful `read` and `download` responses, which contain ONLY the resource's data.
 Every other response MUST have a `data` field at the top level on success, or an `errors` field on failure, but NEVER both.
```

Comment:
Change to: Every JSON response MUST have a `data` field at the top level on success, or an `errors` field on failure.
It MUST NOT have both.